scripts/upload-assemblies.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# This is a generic script for uploading assemblies in the GDK for Unity.
```

Comment:
Add error handling/usage print if the user forgets the PREFIX.
Any requirements as to where this script should be run from?

Reply:
Good call.

And so long as its a Git repo in a SpatialOS project - will add this to the script summary